shared/react-native/ios/Keybase/AppDelegate.m

```diff
@@ -43,6 +45,19 @@ - (BOOL)addSkipBackupAttributeToItemAtPath:(NSString *) filePathString
   return success;
 }
 
+- (void) createBkgReadableDir:(NSString*) path
```

Comment:
That function creates the directory and sets the property on it. That permission bit is inherited by everything in the directory, and Go is going to be the one populating it with all the LevelDB files it is going to create. I can make the name more verbose, but I think it is accurate in what it is doing.